From 151a8353d373df75b346a9d6d6594e27eb66250d Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Sun, 22 May 2016 10:41:13 +0300
Subject: [PATCH 179/239] fix: xdb: update break point at the end of binary
 extension

The old break point was on the end of the SPL, we move
to use BLE in the ATF and this patch update the break point

Change-Id: I55f7bb1d9914b42c016057a3ade31a424c490ecc
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29847
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 scripts/debugger/binary_extension.xdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/debugger/binary_extension.xdb b/scripts/debugger/binary_extension.xdb
index b904ed1..389e23f 100644
--- a/scripts/debugger/binary_extension.xdb
+++ b/scripts/debugger/binary_extension.xdb
@@ -39,7 +39,7 @@ LOAD /SEGMENT /DEBUG /GLOBAL /INIT="" OF @BIN_EXT_ELF_PATH
 END
 
 ! Break at the end of bin_ext_run before jumping back top BootROM
-SET BREAKPOINT AT (0xFFE1C080) HARD
+SET BREAKPOINT AT (0xFFE1C08C) HARD
 RUN
 DELETE BREAKPOINT /ALL
 
-- 
1.9.1

